### PR TITLE
Patch 3

### PR DESCRIPTION
**Pull request by anandkumarcoder**
Added: more cards
fixed : borders
update: hex code to CSS variables

image
image
image
image
image
image
image

### CODE REVIEW

**Review by darshandamre — APPROVED** [review]
good to merge

**Review by AtulHande03a — APPROVED** [review]
The changes look good , you can merge this.

File: Styles/input.css [hunk 1]
```diff
@@ -1,68 +1,64 @@
...
 }
 
-#username:invalid + span::before{
-    content: '✖';
-  color: #ff0000;
-   
+#username + span::before {
+  position: absolute;
+  right: 5px;
+  bottom: 10px;
```
Comment (poojavishnoi) [inline Styles/input.css:20]: try to be consistent in using rem or px. Mostly use rem.

**Review by poojavishnoi — APPROVED** [review]
Looks good to merge

**Pull request merged**